Vagrantfile

```diff
@@ -141,7 +141,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   def name_node(node, name, ec2_instance_name_prefix)
     node.vm.hostname = name
     node.vm.provider :aws do |aws|
-      aws.tags = { 'Name' => ec2_instance_name_prefix + "-" + Socket.gethostname + "-" + name }
+      aws.tags = {
+        'Name' => ec2_instance_name_prefix + "-" + Socket.gethostname + "-" + name,
+        'Jenkins Build' => ENV['BUILD_URL']
```

Comment:
What naming convention do we want to use for tags? Would camel case be simpler perhaps?

Reply:
Sounds good. Did a search and it seems AWS also uses camel case in their examples: https://aws.amazon.com/premiumsupport/knowledge-center/ec2-resource-tags/